dataactvalidator/validation_handlers/file_generation_manager.py

```diff
@@ -9,14 +9,18 @@
 from dataactcore.interfaces.function_bag import mark_job_status
 from dataactcore.models.domainModels import ExecutiveCompensation
 from dataactcore.models.jobModels import Job
-from dataactcore.models.lookups import FILE_TYPE_DICT_LETTER_NAME
 from dataactcore.models.stagingModels import AwardFinancialAssistance, AwardProcurement
 from dataactcore.utils import fileA, fileD1, fileD2, fileE, fileF
 
 from dataactvalidator.filestreaming.csv_selection import write_csv, write_query_to_file
 
 logger = logging.getLogger(__name__)
 
+GEN_FILENAMES = {
```

Comment:
Would recommend extracting the ".csv"'s from these and just add the extension when generating the filename.

Reply:
This further complicates the relatively simple code for the `raw_filename` variable. Because we'd have to add a `.csv` at the end of the filenames before potentially formatting to add the `agency_code`, pulling out the inline `if`/`else` statement as well.